### PR TITLE
Show onboarding screen only once (EXPOSUREAPP-4367)

### PR DESCRIPTION
**Pull request by axelherbstreith**
This PR addresses the persistance handling of the onboaring info screen.
ExposureApp-4152.

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/ContactDiarySettings.kt [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package de.rki.coronawarnapp.submission
```
Comment (BMItter) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/ContactDiarySettings.kt:1]: please recheck package name

Comment (axelherbstreith) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/ContactDiarySettings.kt]: Done

**Review by BMItr — APPROVED** [review]

**Review by Oliver-Zimmerman — APPROVED** [review]

File: Corona-Warn-App/src/main/res/layout/contact_diary_activity.xml [hunk 2]
```diff
@@ -11,7 +11,6 @@
         android:name="androidx.navigation.fragment.NavHostFragment"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        app:defaultNavHost="true"
-        app:navGraph="@navigation/contact_diary_nav_graph" />
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/res/layout/contact_diary_activity.xml:15]: We don't need this?

Comment (axelherbstreith) [reply Corona-Warn-App/src/main/res/layout/contact_diary_activity.xml:15]: Will be set programmatically in the ContactActivity. This allows to dynamically set the first fragment based on the data from the Shared Preferences

Comment (SamuraiKek) [reply Corona-Warn-App/src/main/res/layout/contact_diary_activity.xml:15]: Looks like it works without it but the main activity xml has it so I'm not sure...

**Pull request merged**